go.mod

```diff
@@ -186,6 +200,8 @@ require (
 )
 
 require (
+	github.com/aws/aws-sdk-go-v2/service/cloudformation v1.20.3
```

Comment:
[service/cloudformation/v1.20.4](https://github.com/aws/aws-sdk-go-v2/releases/tag/service%2Fcloudformation%2Fv1.20.4) is the latest?